components/dashboard/src/login/SetupPending.tsx

```diff
@@ -0,0 +1,41 @@
+/**
+ * Copyright (c) 2023 Gitpod GmbH. All rights reserved.
+ * Licensed under the GNU Affero General Public License (AGPL).
+ * See License.AGPL.txt in the project root for license information.
+ */
+
+import { FC } from "react";
+import { Heading2, Subheading } from "../components/typography/headings";
+import cubicleImg from "../images/cubicle.png";
+import cubicleImg2x from "../images/cubicle-2x.png";
+import gitpodIcon from "../icons/gitpod.svg";
+import classNames from "classnames";
+
+type Props = {
+    alwaysShowHeader: boolean;
+};
+export const SetupPending: FC<Props> = ({ alwaysShowHeader }) => {
+    return (
+        <div className="flex-grow flex items-center justify-center p-4">
+            <div className={classNames(alwaysShowHeader ? "" : "lg:hidden", "absolute top-0 left-0 right-0")}>
+                <div className="flex items-center justify-center items-center py-3 space-x-1">
+                    <img src={gitpodIcon} className="w-6 h-6" alt="Gitpod's logo" />
+                    <span className="text-lg">Gitpod</span>
+                </div>
+            </div>
+            <div className="max-w-md flex flex-col items-center justify-center text-center">
+                <img
+                    className="mb-8"
+                    src={cubicleImg}
+                    srcSet={`${cubicleImg} 1x, ${cubicleImg2x} 2x`}
```

Comment:
**issue**: The problem with going with the cube is that we would need to have a different version for the dark theme as the default version looks a bit jarring. From a [relevant comment](https://linear.app/gitpod/issue/WEB-308#comment-f2ff5404) in WEB-308:
> We've previously used a property in src to switch between two illustrations for light and dark theme (see [relevant PR](https://github.com/gitpod-io/gitpod/pull/4981/files)) but in this case the honeycomb illustration is using CSS to source the background (see [relevant diff](https://github.com/gitpod-io/gitpod/blob/0965bc98a84038f537026787210c2d03c2e9868b/components/dashboard/src/dedicated-setup/styles.css#L9)).

<img width="2974" alt="Frame 1471" src="https://github.com/gitpod-io/gitpod/assets/120486/7fa96235-b00c-497f-b1da-93a8b0b47277">

Reply:
Sounds good, @selfcontained. You call—it's not blocking. In case you want to add this, here's **[the relevant frame](https://www.figma.com/file/VL0CJpHqcDJVJ4BEkoyRFV/Gitpod-Website-Component-Library?type=design&node-id=1772%3A3079&t=Ib0Jnfbeh4rNZlDR-1)** from the dark themed cube which we already use in `www.gitpod.io`. 🏓 

Cc @chrifro

Reply:
Thanks for the ping @gtsiolis 🙌 If we use this image, we should also use the dark theme version.

**Some more thoughts:**
- choice of image: the image cube symbolizes a workspace that has everything at hand and is ready to use. This contradicts the messaging. Alternatively, we could either use an empty room which symbolizes that things are not ready yet [[1](https://www.figma.com/file/nhTLkTz28LMCxQ7ltuH10H/Admin-onboarding-%23UX-2?type=design&node-id=3752%3A3489&t=kHxFXqzwPD4vVSJd-1)] or use this image with an adjusted bubble text [[1](https://www.figma.com/file/nhTLkTz28LMCxQ7ltuH10H/Admin-onboarding-%23UX-2?type=design&node-id=3653%3A3275&t=kHxFXqzwPD4vVSJd-1)]
- right now the visual hierarchy of text and image is relatively even. That makes it tricky for the eyes to know what to focus on first. I'd suggest increasing the size of the image
- wording: "This instance of Gitpod is not quite ready" sounds like it's a Gitpod issue, but actually, the status is blocked by the admin. I'd remove the first sentence and potentially add a button to nudge the admin

WDYT @selfcontained :)